src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,20 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer bulldozer1 = new Bulldozer(300, "Heavy");
+        Bulldozer bulldozer2 = new Bulldozer(120, "Light");
+        Bulldozer bulldozer3 = new Bulldozer(200, "Digging");

```

Comment:
```suggestion
        Bulldozer bulldozer1 = new Bulldozer(300, "Heavy");
        Bulldozer bulldozer2 = new Bulldozer(120, "Light");
        Bulldozer bulldozer3 = new Bulldozer(200, "Digging");

```

It's bad naming style when we use numbers.